src/error.rs

```diff
@@ -1930,7 +1930,7 @@ impl IntoDiagnostics<FileId> for TypecheckError {
                         .with_message("multiple rows declaration")
                         .with_labels(mk_expr_label(&span_opt))
                         .with_notes(vec![
-                            format!("Found an expression of a record type with the row `{}: {}`", ident, conflict.as_ref().cloned().unwrap()),
+                            format!("Found an expression of a record type `{}` with the row `{}`", conflict.as_ref().cloned().unwrap(), ident),
```

Comment:
With the change it shows up like this:
```
error: multiple rows declaration
  ┌─ repl-input-4:1:89
  │
1 │ let f | forall r. { ; r } -> { x : Number; r } = fun r => %record_insert% "x" r 1 in (f { x = 0 }: _)
  │                                                                                         ^^^^^^^^^ this expression
  │
  = Found an expression of a record type `{ x: _a }` with the row `x`
  = But this type appears inside another row type, which already has a declaration for the field `x`
  = A type cannot have two conflicting declarations for the same row
```

Reply:
Ok, I'm not 100% sure of what really is `conflict` right now. It might actually be the tail of the original record type, but fixing an error message later on if we find edge cases is not very costly, so let's proceed with that :+1: